src/widgets/SingleLineTextInput.js

```diff
@@ -85,7 +85,6 @@ export default class SingleLineTextInput extends createjs.Container {
     this.accessible.placeholder = placeholderText;
 
     this.addEventListener('click', this.onFocus);
-    // todo: handle mouse based blur causing event
```

Comment:
Nit: technically the todo's in this file should remain since fixing text input by moving focus to the DOM translation element will be a separate PR.

Reply:
reverted